Author: Okinea Dev <hi@okinea.dev>
Description: Removed partner ID between Xiaomi and OneDrive

diff --git a/images/super/mi_ext_a/etc/build.prop b/images/super/mi_ext_a/etc/build.prop
index 0000000..0000000 100755
--- a/images/super/mi_ext_a/etc/build.prop
+++ b/images/super/mi_ext_a/etc/build.prop
@@ -1,3 +1,2 @@
-ro.microsoft.onedrive_partner_code=/mi_ext/product/etc/onedrive_token.txt
 ro.product.mod_device=fog_global
 ro.miui.build.region=global
diff --git a/images/super/mi_ext_a/product/etc/onedrive_token.txt b/images/super/mi_ext_a/product/etc/onedrive_token.txt
deleted file mode 100755
index 0000000..0000000
--- a/images/super/mi_ext_a/product/etc/onedrive_token.txt
+++ /dev/null
@@ -1 +0,0 @@
-xiaomipreinstallota
